pep-0517.txt

```diff
@@ -166,48 +166,71 @@ The build backend object is expected to have attributes which provide
 some or all of the following hooks. The common ``config_settings``
 argument is described after the individual hooks::
 
-  def get_build_requires(config_settings):
+  def build_sdist(source_directory, sdist_directory, config_settings):
+      ...
+
+MUST build an unpacked sdist from the ``source_directory`` and place it into the
+specified ``sdist_directory``. If the ``source_directory`` is already an
+unpacked sdist (as opposed to a VCS source tree) then it SHOULD simply copy the
```

Comment:
A couple more cases occurred to me:
- It's a known VCS checkout but the VCS tools are not available (e.g. `git` is not on PATH)
- It's a VCS checkout of a VCS the tool does not recognise.